tests/test_alignment_from_everything.yml

```diff
@@ -0,0 +1,42 @@
+- name: Run alignment to bam fastq and spring files to bam files
+  command: nextflow run main.nf -profile test,alignment_from_everything --outdir results --save_mapped --save_output_as_bam
+  tags:
+    - alignment_from_everything
+  files:
+    - path: results/csv/mapped.csv
+      md5sum: 5a10f49a6a691c84e31e7dd8e91c8201
+    - path: results/csv/markduplicates.csv
+      md5sum: 293ae6ec0286272470dd8d6edf4b4fc9
+    - path: results/csv/markduplicates_no_table.csv
+      md5sum: e379af37e14b94c17465654f971bd23f
+    - path: results/csv/recalibrated.csv
+      md5sum: 976529de568cdf201e6b9dfb2da3f62f
+    - path: results/multiqc
+    - path: results/preprocessing/mapped/test/test.sorted.bam
+    - path: results/preprocessing/mapped/test/test.sorted.bam.bai
+    - path: results/preprocessing/mapped/test2/test2.sorted.bam
+    - path: results/preprocessing/mapped/test2/test2.sorted.bam.bai
+    - path: results/preprocessing/mapped/test3/test3.sorted.bam
+    - path: results/preprocessing/mapped/test3/test3.sorted.bam.bai
+    - path: results/preprocessing/mapped/test_bam/test_bam.sorted.bam
+    - path: results/preprocessing/mapped/test_bam/test_bam.sorted.bam.bai
+    - path: results/preprocessing/markduplicates/test/test.md.bam
+    - path: results/preprocessing/markduplicates/test/test.md.bam.bai
+    - path: results/preprocessing/markduplicates/test2/test2.md.bam
+    - path: results/preprocessing/markduplicates/test2/test2.md.bam.bai
+    - path: results/preprocessing/markduplicates/test3/test3.md.bam
+    - path: results/preprocessing/markduplicates/test3/test3.md.bam.bai
+    - path: results/preprocessing/markduplicates/test_bam/test_bam.md.bam
+    - path: results/preprocessing/markduplicates/test_bam/test_bam.md.bam.bai
+    - path: results/preprocessing/recal_table/test/test.recal.table
+    - path: results/preprocessing/recal_table/test2/test2.recal.table
+    - path: results/preprocessing/recal_table/test3/test3.recal.table
+    - path: results/preprocessing/recal_table/test_bam/test_bam.recal.table
+    - path: results/preprocessing/recalibrated/test/test.recal.bam
+    - path: results/preprocessing/recalibrated/test/test.recal.bam.bai
+    - path: results/preprocessing/recalibrated/test2/test2.recal.bam
+    - path: results/preprocessing/recalibrated/test2/test2.recal.bam.bai
+    - path: results/preprocessing/recalibrated/test3/test3.recal.bam
+    - path: results/preprocessing/recalibrated/test3/test3.recal.bam.bai
+    - path: results/preprocessing/recalibrated/test_bam/test_bam.recal.bam
+    - path: results/preprocessing/recalibrated/test_bam/test_bam.recal.bam.bai
```

Comment:
Any other files that need to be added to the test?

Reply:
No I'd say we're good